drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -36,13 +36,47 @@ def self.build(node_class, field_permissions: nil, transform_changes: nil)
     field :id, ID, null: false
     field :record_id, ID, null: false, method: :item_id
     field :record_name, String, null: false
+    field :graphql_type, String, null: false
     field :event, HmisSchema::Enums::AuditEventType, null: false
     field :created_at, GraphQL::Types::ISO8601DateTime, null: false
     field :user, Application::User, null: true
     field :object_changes, Types::JsonObject, null: true, description: 'Format is { field: { fieldName: "GQL field name", displayName: "Human readable name", values: [old, new] } }'
 
+    available_filter_options do
+      arg :audit_event_record_type, [ID]
+      arg :user_id, [ID]
```

Comment:
For consistency, can we call this `arg :user, [ID]`? That's what we do for other ID selection, and we want it to show up in the UI as "User" not "User ID"